Library/Formula/ipmiutil.rb

```diff
@@ -2,12 +2,12 @@
 
 class Ipmiutil < Formula
   homepage 'http://ipmiutil.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/project/ipmiutil/ipmiutil-2.9.1.tar.gz'
-  sha1 'f23fabe8339842fea9b8c2a601717dc002e44a9d'
+  url 'https://downloads.sourceforge.net/project/ipmiutil/ipmiutil-2.9.5.tar.gz'
+  sha1 '265f022c876da373b2ecb4be2bc0f98e65f70977'
 
-  # Make ipmiutil treat Darwin as BSD
-  patch :DATA
+  depends_on "openssl"
 
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!